rendermodules/pointmatch/swap_point_match_collections.py

```diff
@@ -0,0 +1,97 @@
+import renderapi
+from rendermodules.module.render_module import RenderModule, RenderModuleException
+from rendermodules.pointmatch.schemas import SwapPointMatches, SwapPointMatchesOutput
+import time
+
+example = {
+    "render": {
+        "host": "em-131db",
+        "port": 8080,
+        "owner": "TEM",
+        "project": "17797_1R",
+        "client_scripts": "/allen/aibs/pipeline/image_processing/volume_assembly/render-jars/production/scripts/"
+    },
+    "match_owner": "match_owner",
+    "source_collection": "source_collection",
+    "target_collection": "target_collection",
+    "zValues": []
```

Comment:
does an empty list mean do nothing? or swap them all? just some documentation, this example seems to imply to me subtly that an empty list will do them all, because why show an example that will do nothing.